### ISSUE TITLE
chip.select doesn't work ?

### ISSUE THREAD

**Issue by bibislayer** [Dogfalo/materialize#4037]
### Description
Event chip.select doesn't working

### Screenshots / Codepen
http://codepen.io/anon/pen/mOZMQW?editors=1001

**Comment by arpit1997** [comment]
There is a small mistake in your code @bibislayer change class `chips`  to `chip`
```
 <body>
      <div class="chip">Hello</div>
      <div class="chip">
    Tag
    <i class="close material-icons">close</i>
  </div>
      <script>
        $(document).ready(function() {
          $('.chips').material_chip();
          $('.chips').on('chip.add', function(e, chip){
            console.log("Added",chip);
          });
          $('.chips').on('chip.delete', function(e, chip){
            console.log("Delete",chip);
          });
           $('.chips').on('chip.select', function(e, chip){
            console.log("Select",chip);
          });
        });
      </script>
  </body
```
Follow the docs [here](http://materializecss.com/chips.html)

**Comment by bibislayer** [comment]
My code haven't mistake, 
Retry http://codepen.io/anon/pen/mOZMQW?editors=1001
I juste remove the .chips-initial now it's a copy/past from http://materializecss.com/chips.html

**Comment by arpit1997** [comment]
@bibislayer The last code i have seen, there was a mistake. You did not initilize you tags like this one
```
  $('.chips-initial').material_chip({
    data: [{
      tag: 'Apple',
    }, {
      tag: 'Microsoft',
    }, {
      tag: 'Google',
    }],
  });
```
So i fixed that mistake first.
No chip was showing before in the last code

**Comment by bibislayer** [comment]
i already correct this, just i don't use initial, but you could add a tag and remove this on the input at the top.

**Comment by bibislayer** [comment]
now Initial chips is added and chip.select event never work.

**Comment by acburst** [comment]
This is indeed a bug, thanks for pointing this out.

Fixed in 84f204d
